tests/test-cases/Group11-Upgrade/11-01-Upgrade.md

```diff
@@ -17,9 +17,11 @@ This test requires that a vSphere server is running and available
 7. Roll back to the previous version
 8. Upgrade again to the upgraded version
 9. Check the previous created container and image are still there
+10. Check the previous created container's display name and datastore folder name
 
```

Comment:
suggest to add case to check new container after upgrade will have different display name